index.js

```diff
@@ -18,9 +18,7 @@ const cache = new QuickLru({maxSize: 100_000});
 const isObject = value =>
 	typeof value === 'object'
 		&& value !== null
-		&& !(value instanceof RegExp)
-		&& !(value instanceof Error)
-		&& !(value instanceof Date);
+		&& (Array.isArray(value) || Object.getPrototypeOf(value) === Object.prototype);
```

Comment:
Move `Object.getPrototypeOf(value) === Object.prototype` out into a const function to improve readability by giving it a name.

Reply:
And I think the behavior should be documented.